mdal/mdal_data_model.hpp

```diff
@@ -267,6 +267,12 @@ namespace MDAL
       std::string crs() const;
       size_t faceVerticesMaximumCount() const;
 
+      std::string getMetadata( const std::string &key );
```

Comment:
we do not use getXYZ() syntax for getters, can you rename to `std::string metadata(...)` ?

Reply:
I actually copied the structure and names exactly from DatasetGroup - including the getter - for complete consistency.

I cannot rename to `metadata` - because there is already a class member Mesh::metadata (and DatasetGroup::metadata since the former is an exact copy of the latter in structure).

The `getMetadata()` method does add the ability to get metadata by key as opposed to by index. In the DatasetGroup case this is used in one place  DAT driver.

I can rename the method anything you want (except metadata) but I do strongly suggest that we keep the signatures for Mesh metadata and DatasetGroup metadata consistent, otherwise we are just creating work and potential errors.

Reply:
ok